src/safeposix/syscalls/sys_constants.rs

```diff
@@ -15,6 +15,8 @@ pub const DEFAULT_GID : u32 = 1000;
 
 // RESOURCE LIMITS
 
+pub const SIGNAL_MAX : i32 = 64;
```

Comment:
Since we only have 32 signals and even in Lind-GlibC we are checking for 32 signals only, can we set this to 64?

Reply:
And also update SigSetType to be `u32` instead of `u64`?

Reply:
We reserve the upper 32 bits of `sigset` in case we want to implement real-time signals in the future.